### PR TITLE
Update the description in script

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Update the description in script as follows:
update-generated-completions.sh
update-generated-docs.sh

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: hack/update-generated-completions.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-# This script sets up a go workspace locally and builds all go components.
+# This script sets up a go workspace locally and generates the binary completed.
```
Comment (stevekuznetsov) [inline hack/update-generated-completions.sh]: This script doesn't generate binaries, rather it generates shell auto-completion scripts.

Comment (yuexiao-wang) [inline hack/update-generated-completions.sh]: Got it and I will apply it to this commit.
Thanks!

Comment (yuexiao-wang) [inline hack/update-generated-completions.sh]: @stevekuznetsov PTAL
It has done to update this PR.
Thanks!

**Comment by stevekuznetsov** [comment]
Thanks for the comment fixes! [merge]

**Comment by yuexiao-wang** [comment]
@stevekuznetsov PTAL
This PR is merged failed  by openshift-bot.
Thanks!

**Comment by stevekuznetsov** [comment]
@openshift-bot, the [last build](https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/8272/) failed from the following flakes:
 - [check]() failed on: https://github.com/openshift/origin/issues/10487

re[merge]

**Comment by stevekuznetsov** [comment]
@openshift-bot, the [last build](https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/8378/) failed from the following flakes:
 - [conformance]() failed on: https://github.com/openshift/origin/issues/9548 https://github.com/openshift/origin/issues/9490

re[merge]

**Pull request merged**